### PR TITLE
4383 dod

### PR DESCRIPTION
**Pull request by matthopson**
Note: npm update was clean (due to it just being run on develop branch)

### CODE REVIEW

File: shared/src/utilities/JoiValidationDecorator.js [hunk 1]
```diff
@@ -186,10 +186,10 @@ exports.joiValidationDecorator = function(
   entityConstructor.prototype.toRawObjectFromJoi = toRawObjectPrototype;
 
   entityConstructor.validateRawCollection = function(
-    collection = [],
+    collection,
     { applicationContext },
   ) {
-    return collection.map(entity =>
+    return (collection || []).map(entity =>
```
Comment (kkoskelin) [inline shared/src/utilities/JoiValidationDecorator.js:192]: Tell me more about this.  Personal preference? nbd, just curious.

**Review by kkoskelin — APPROVED** [review]

File: shared/src/utilities/JoiValidationDecorator.js (hunk 1 above)
Comment (matthopson) [reply shared/src/utilities/JoiValidationDecorator.js:192]: SonarCloud didn't like it ¯\_(ツ)_/¯ 
https://sonarcloud.io/project/issues?branch=develop&id=EF-CMS-SHARED&open=AXA6lgM0r3puLwSqvyxo&resolved=false&types=CODE_SMELL

**Review by rachaelparris — APPROVED** [review]

File: shared/src/utilities/JoiValidationDecorator.js (hunk 1 above)
Comment (kkoskelin) [reply shared/src/utilities/JoiValidationDecorator.js:192]: Good to know! Thanks.

Comment (matthopson) [reply shared/src/utilities/JoiValidationDecorator.js:192]: Right? I thought this looked perfectly fine.

**Pull request merged**